### PR TITLE
feat(riseupvpn): fill test keys

### PR DESCRIPTION
**Pull request by bassosimone**
Part of https://github.com/ooni/probe-engine/issues/1060.

Code adapted from https://github.com/ooni/probe-cli/pull/171.

### CODE REVIEW

**Comment by bassosimone** [comment]
@cyBerta we moved the code to generate test keys from probe-cli to probe-engine. I adapted your code in https://github.com/ooni/probe-cli/pull/171. Does the code look good to you? Thanks 🙏 !

File: experiment/riseupvpn/riseupvpn.go [hunk 1]
```diff
@@ -285,10 +286,23 @@ func NewExperimentMeasurer(config Config) model.ExperimentMeasurer {
 // Note that this structure is part of the ABI contract with probe-cli
 // therefore we should be careful when changing it.
 type SummaryKeys struct {
-	IsAnomaly bool `json:"-"`
+	APIBlocked      bool `json:"riseupvpn_api_blocked"`
+	ValidCACert     bool `json:"riseupvpn_ca_cert_valid"`
+	FailingGateways int  `json:"riseupvpn_failing_gateways"`
```
Comment (bassosimone) [inline experiment/riseupvpn/riseupvpn.go:291]:
```suggestion
	APIBlocked      bool `json:"api_blocked"`
	ValidCACert     bool `json:"valid_ca_cert"`
	FailingGateways int  `json:"failing_gateways"`
```

Let's remove the `riseupvpn` prefix also from here.

**Comment by cyBerta** [comment]
hi @bassosimone sorry for being late. I was planning to check the PR. Anyways thanks for your work!!

**Comment by bassosimone** [comment]
@cyBerta no worries! If you can, maybe take a look at the diff anyway? cheers!

**Pull request merged**